governance-services/latest/install/uninstall.md

```diff
@@ -33,12 +33,12 @@ $ java -jar bin\alfresco-mmt.jar uninstall <ModuleId> <WARFileLocation>
     ```bash
     Module 'org_alfresco_module_rm' installed in 'webapps/alfresco'
        -    Title:        AGS Repo
-       -    Version:      7.1
+       -    Version:      7.2
        -    Install Date: null
        -    Description:   Alfresco Governance Services Repository Extension
     Module 'alfresco-rm-enterprise-repo' installed in 'webapps/alfresco'
        -    Title:        AGS Enterprise Repo
-       -    Version:      7.1
+       -    Version:      7.2
        -    Install Date: Tue Sept 28 08:50:15 UTC 2021
```

Comment:
_Future update_: we should consider making lines 42 & 60 more generic or update the date